README.md

```diff
@@ -7,38 +7,38 @@
 **Kover** - Gradle plugin for Kotlin code coverage tools: [Kover](https://github.com/JetBrains/intellij-coverage)
 and [JaCoCo](https://github.com/jacoco/jacoco).
 
-Minimal supported `Gradle` version: `6.6`.
+Minimum supported version of `Gradle` is `6.6`.
```

Comment:
Couldn't find any credible sources for this one, but googling for "Minimum version" yield way more results than vs "Minimal version", so `Minimum` _seems_ to be the correct word in this context. But both sound kinda weird to my ear tbh